### DISCUSSION TITLE
<web3button>

### DISCUSSION THREAD

**Discussion by indasashko123** [WalletConnect/walletconnect-monorepo#2582] [Developer Support]
The web3button button is not displayed on some devices. How to fix it? What requirements should the device have?

**Comment by indasashko123** [comment]
I use react

**Comment by glitch-txs** [comment]
Hi, in which devices? Do you have a reproduction example?

**Reply by indasashko123** [reply]
huawei p smart fig-lx1, 
Iphone (OS update solved the problem), 
In fact, there were many cases. Code to show?

**Reply by glitch-txs** [reply]
Yes, that'd be great! :)

Also, does this happen in labs as well? These are official integrations

https://lab.web3modal.com/

By displayed you mean the button or the modal when clicking the button?
